exporter/loadbalancingexporter/exporter.go

```diff
@@ -146,7 +148,11 @@ func (e *exporterImp) addMissingExporters(ctx context.Context, endpoints []strin
 			cfg := e.buildExporterConfig(endpoint)
 			exp, err := e.exporterFactory.CreateTracesExporter(ctx, e.templateCreateParams, &cfg)
 			if err != nil {
-				e.logger.Warn("failed to create new trace exporter for endpoint", zap.String("endpoint", endpoint))
+				e.logger.Error("failed to create new trace exporter for endpoint", zap.String("endpoint", endpoint), zap.Error(err))
+				continue
+			}
+			if err = exp.Start(ctx, e.host); err != nil {
```

Comment:
🤦🏽 Great catch!

Reply:
that's why enabling sending_queue will prevent spans from being reported. 👍🏽